lib/generators/rolemodel/saas/devise/devise_generator.rb

```diff
@@ -124,6 +126,19 @@ def modify_existing_files
           RUBY
         end
       end
+
+      def define_devise_mailer_layout
+        append_to_file 'config/initializers/devise.rb', <<~RUBY
+
+          Rails.application.config.to_prepare do
+            Devise::Mailer.layout 'mailer'
```

Comment:
In C12 we just uncommented and changed a line to ApplicationMailer https://github.com/RoleModel/c12_core/commit/ce81e99f4e1471256c1d4a279ff6e37ba3f3b71e#diff-cb6e8126296dbc007e7625eee863de5c3213ed949b9999ea3418775f65826531R66